src/mixins/widget.js

```diff
@@ -3,14 +3,22 @@ import { warn } from '../util/warn';
 export const createWidgetMixin = ({ connector } = {}) => ({
   inject: {
     instantSearchInstance: {
-      name: 'instantSearchInstance',
+      // TODO: should we prevent name clashes like this?
+      // from: '$ais_instantSearchInstance',
       default() {
         const tag = this.$options._componentTag;
         throw new TypeError(
           `It looks like you forgot to wrap your Algolia search component "<${tag}>" inside of an "<ais-instant-search>" component.`
         );
       },
     },
+    indexWidget: {
+      // from: '$ais_indexWidget',
```

Comment:
can we rename it? I prefer to use `indexWidget` inside the mixin, but `$ais_indexWidget` as real provided value, but the from didn't seem to work in my testing. cc @posva is that expected?

Reply:
it should work, yeah. If you reproduce it on a jsfiddle, open an issue on vue. It could also be a bug on the test utils not working (not sure if that's what you meant by testing)

Reply:
I can no longer reproduce this, don't know why it initially didn't work